### PR TITLE
Fix ufm plugin

### PR DESCRIPTION
**Pull request by Mmduh-483**
Fix failure when send multi guids to ufm.
Add building plugins to the image.
Fix loading plugin error built with different version.
Fix typo in http client.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -61,7 +61,7 @@ build: $(BUILDDIR)/$(BINARY_NAME) ; $(info Building $(BINARY_NAME)...) ## Build
 	$(info Done!)
 
 $(BUILDDIR)/$(BINARY_NAME): $(GOFILES) | $(BUILDDIR)
-	@cd $(BASE)/cmd/$(BINARY_NAME) && CGO_ENABLED=0 $(GO) build -o $(BUILDDIR)/$(BINARY_NAME) -tags no_openssl -v
+	@cd $(BASE)/cmd/$(BINARY_NAME) && CGO_ENABLED=1 $(GO) build -o $(BUILDDIR)/$(BINARY_NAME) -tags no_openssl -v
```
Comment (moshe010) [inline Makefile]: I assume you need this to load the plugin right?

File: Makefile [hunk 2]
```diff
@@ -90,7 +90,7 @@ plugins: noop-plugin ufm-plugin  ; $(info Building plugins...) ## Build plugins
 
 %-plugin: $(PLUGINSBUILDDIR)
 	@echo Building $* plugin
-	$Q $(GO) build -ldflags "-X $(REPO_PATH)/version=1.0" -o $(PLUGINSBUILDDIR)/$*.so -buildmode=plugin -i $(REPO_PATH)/pkg/sm/plugins/$*
+	$Q CGO_ENABLED=1 $(GO) build -ldflags "-X $(REPO_PATH)/version=1.0" -o $(PLUGINSBUILDDIR)/$*.so -buildmode=plugin -i $(REPO_PATH)/pkg/sm/plugins/$*
```
Comment (moshe010) [inline Makefile]: please create global variable to set the CGO_ENABLED so we don't need to change it in every command

File: Makefile (hunk 1 above)
Comment (Mmduh-483) [reply Makefile]: yes

File: Makefile (hunk 2 above)
Comment (Mmduh-483) [reply Makefile]: added

**Pull request merged**